subgraphs/spotify/src/resolvers/Developer.ts

```diff
@@ -0,0 +1,6 @@
+import { getFieldConfigs } from '../fieldConfigs';
+import { DeveloperResolvers } from '../__generated__/resolvers-types';
+
+export const Developer: DeveloperResolvers = {
```

Comment:
Eventually this stuff might make sense in its own subgraph 🙂. This is specifically for our version of the app where we can add synthetic timeouts and errors. This stuff is not available in the Spotify API as its specific to this demo app.

No need to worry about it in this PR since its already massive, but something to consider for a future PR.

Reply:
Good callout, I created #67 to track that and make it happen. I think it's an easy add on with another Netlify function.